bagel/dictionary_models.py

```diff
@@ -76,6 +98,7 @@ class IdentifierNeurobagel(Neurobagel):
 class ToolNeurobagel(Neurobagel):
     """A Neurobagel annotation for an assessment tool column"""
 
+    # TODO: Why is this optional again?
```

Comment:
Good question. I remember that in the beginning we spent a lot of thought on how not to have a "greedy" validator just always run through the past of least resistance, i.e. by just validating everything as `participant_id` column (iirc). I also can't see a good reason how this is optional.

Reply:
Maybe remove the optional statement and see if any tests break as a first sniff test?

Reply:
Good idea. No tests fail when I remove the `Optional`...

After some digging I've found that I was actually the one who initially added this class 👀, in https://github.com/neurobagel/bagel-cli/pull/166/files, but there was no special comment or anything about the `Optional` part. Turns out I was also the culprit for the `identifies: "str"` instead of `identifies: str`, in the same PR - so I'm gonna chalk it up to me being silly that day and just writing some wacky type hints.

I'll remove it for now but update the note to refer to this PR, in case we discover some weird behaviour and need to revert this.